src/pages/host/getHostEventString.tsx

```diff
@@ -273,15 +308,17 @@ export const HostEventLog: React.FC<{
   logs: any;
   isCode: boolean;
 }> = ({ title, logs, isCode }) => (
-  <StyledCollapse bordered={false}>
-    <Panel header={title} key="1">
-      {{ isCode } ? (
-        <Code multiline={false} language="shell">
-          {logs}
-        </Code>
-      ) : (
-        { logs }
-      )}
-    </Panel>
-  </StyledCollapse>
+  <span data-cy="host-event-logs">
+    <StyledCollapse bordered={false}>
```

Comment:
is it possible to add a data-cy attribute to StyledCollapse instead of nesting it inside of a span? this will make our HTML cleaner especially if the the outer span's primary purpose is to hold a data-cy attribute